samwise.js

```diff
@@ -1,5 +1,5 @@
 /**
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!